wordpress-mobile/WordPress-iOS
# Post excerpt: increase minimum height

Fixes #9579 

This change increases the minimum field height so it’s more obviously a multi-line field when blank. This actually affects three fields:
- Post > Post Settings > Excerpt > Excerpt entry
- Site > People > + > Message field > Message entry
  - Note there is an outstanding [issue](https://github.com/wordpress-mobile/WordPress-iOS/issues/10795) with this message not being fully displayed on the invitation view.
- Me > My Profile > About Me > text entry

To test:
- Go to one of the views listed above.
- Clear the text, if any.
- Verify the empty text field now appears about three lines high.

<img width="300" alt="empty_text" src="https://user-images.githubusercontent.com/1816888/51000557-bd291600-14ea-11e9-9fdf-b4f425d302fc.png">

Update release notes:
